docs/maintainers/ExampleDocumentationClass.m

```diff
@@ -0,0 +1,65 @@
+classdef ExampleDocumentationClass < handle
```

Comment:
Do we really need this? I think pointing users to a class like Timeline was just fine.  There's a whole lot of text here that no one will ever read.   Was this aimed at a 'maintainer' or a 'contributer'?  The former would be familiar with the current code anyway, and the latter probably wouldn't read this when making a small change and most likely wouldn't be adding a whole new class.

At the end of the day everything in this example file has already been explained in words in the CONTRIBUTING file, and is demonstrated well enough in e.g. Timeline.m.  Really details such as error id formats, etc. will always just be changed by one of us after a PR is opened by a 'contributor' or explained by one of us in person to a new 'maintainer'.  I maintain that the contributing file is comprehensive enough and would like to avoid so many 'docs about docs'.

Reply:
I think meta-documentation is important, but I agree that it shouldn't be ott. If we still disagree about this after we discuss tomorrow, maybe we should poll others.